src/zenml/enums.py

```diff
@@ -30,6 +30,15 @@ class ArtifactType(StrEnum):
     BASE = "BaseArtifact"
 
 
+class VisualizationType(StrEnum):
+    """All possible visualization types."""
```

Comment:
```suggestion
class VisualizationType(StrEnum):
    """All possible built-in visualization types."""
```

Reply:
`built-in` sounds wrong IMO because this is not really extensible (each type has hardcoded behavior in the frontend). Changed it to 'All currently available visualization types.' instead.